README.md

```diff
@@ -93,7 +77,7 @@ following steps:
 6.  Remove `"type": "module"` from `package.json` so that we can have
```

Comment:
Total nice-to-have: is this something we can automate? It's not a big imposition, but doing this manually feels like something that a more established framework would take care of for us.

Reply:
this is a dumb annoyance with this repo where it's created with vite but we also run koa within it - ideally in the future we have an `npx create hatchify-app --template koa-react` or something and it automates the entire setup process